assets/js/googlesitekit/modules/create-info-store.test.js

```diff
@@ -30,6 +30,7 @@ import { STORE_NAME as CORE_SITE } from '../datastore/site/constants';
 import { STORE_NAME as CORE_USER } from '../datastore/user/constants';
 
 const MODULE_SLUG = 'test-slug';
+const TEST_STORE_NAME = 'test/' + MODULE_SLUG;
```

Comment:
Minor detail, but let's use the interpolated template string syntax instead.

Reply:
Marked as resolved but still unaddressed.

Reply:
[Addressed here](https://github.com/google/site-kit-wp/pull/2525/files#diff-909552a7f570a1ec29e503680ef0539b36b2a060ea2c81f324b530caf8989352R32)